src/transport_ofi.c

```diff
@@ -555,6 +553,19 @@ void shmem_transport_ofi_stx_allocate(shmem_transport_ctx_t *ctx)
     return;
 }
 
+
+shmem_transport_addr_t shmem_transport_get_local_addr(void)
+{
+    shmem_transport_addr_t addr;
+
+    void *addr_ptr = shmem_transport_ofi_info.fabrics->src_addr;
```

Comment:
OFI says "a provider shall return an address that corresponds to the indicated fabric, domain, node, and/or service fields".  If this can correspond to the specific domain, then I'm not sure it will serve as a node name.

@sayantansur Is this the right way to get a node name from OFI?